packages/schematics/package.json

```diff
@@ -1,21 +1,42 @@
 {
   "name": "@nrwl/schematics",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Nrwl Extensions for Angular: Schematics",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "RxJS",
+    "Angular",
+    "Workspace",
+    "NgRx",
+    "Schematics",
+    "Angular CLI"
+  ],
   "bin": {
     "create-nx-workspace": "./bin/create-nx-workspace.js",
     "nx": "./src/command-line/nx.js"
   },
   "main": "index.js",
   "types": "index.d.js",
-  "peerDependencies": {},
+  "peerDependencies": { },
   "author": "Victor Savkin",
   "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "homepage": "https://github.com/nrwl/nx#readme",
   "schematics": "./src/collection.json",
   "dependencies": {
+    "@ngrx/schematics": "^5.2.0",
+    "@schematics/angular": "^0.4.6",
```

Comment:
ping it to a particular version. `@schematics/angular` isn't super stable, so I'd like our users to have the version with test with.

Reply:
done.